packages/contracts-bedrock/src/cannon/MIPS.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity 0.8.15;
 import { ISemver } from "src/universal/ISemver.sol";
 import { IPreimageOracle } from "./interfaces/IPreimageOracle.sol";
 import { PreimageKeyLib } from "./PreimageKeyLib.sol";
+import "src/cannon/libraries/MIPSInstructions.sol";
```

Comment:
The solidity docs describe this as an anti-pattern [here](https://docs.soliditylang.org/en/v0.8.15/layout-of-source-files.html#syntax-and-semantics):
> This form is not recommended for use, because it unpredictably pollutes the namespace. If you add new top-level items inside “filename”, they automatically appear in all files that import like this from “filename”. It is better to import specific symbols explicitly. 

So, should this import instead be something like?:
```suggestion
import * as instructions from "src/cannon/libraries/MIPSInstructions.sol";
```

Reply:
Yeah good callout. This is good hygiene (though in our case we'll most likely be using all functions).

Reply:
cc @clabby because he introduced this style to the codebase